smart-contracts/contracts/airdrop/src/admin.rs

```diff
@@ -0,0 +1,243 @@
+use std::string::String;
+
+use cosmwasm_std::{DepsMut, Env, Response, StdError, Uint128};
+use cw20_base::contract::query_balance;
+use cw_asset::Asset;
+
+use crate::helpers::get_total_in_user_info;
+use crate::state::{AirdropConfig, AIRDROP_CONFIG, USER_INFO};
+use crate::AirdropErrors;
+
+pub fn execute_update_airdrop_config(
+    deps: DepsMut,
+    env: Env,
+    config: AirdropConfig,
+) -> Result<Response, AirdropErrors> {
+    // Check various conditions to validate the airdrop configuration update
+
+    // Check if the start height and end height are not zero,
+    // indicating a valid airdrop window
+    if config.start_height != 0 && config.end_height != 0 {
```

Comment:
Is this check needed together with the check on line 23?

Reply:
Changed the checks a little as I wrote them in a hurry.